autopush/router/simple.py

```diff
@@ -48,6 +49,13 @@ def __init__(self, ap_settings, router_conf):
         """Create a new SimpleRouter"""
         self.ap_settings = ap_settings
         self.metrics = ap_settings.metrics
+        self.conf = router_conf
+        self.waker = None
+        if self.conf is not None and self.conf.get("server") is not None:
+            # This should use a Wake resolver dict, but there's only one
+            # right now, so skip a few instructions.
+            self.waker = UDPWake(host=self.conf.get("server"),
+                                 cert=self.conf.get("cert"))
```

Comment:
We create a waker on every single simplepush (and only simplepush?) notification even though only UDP wake clients need it?